data/kramdown-rfc2629.erb

```diff
@@ -49,8 +49,8 @@
 <%    venue = KramdownRFC::ParameterSet.new(venue) -%>
     <note title="About This Document" removeInRFC="true">
       <t>
-<%   if editors = venue[:editors] -%>
-        The editor's copy of this draft can be found at <eref target="<%=editors%>"/>.
+<%   if latest = venue[:latest] -%>
+        The editor's copy of this draft can be found at <eref target="<%=latest%>"/>.
 <%   end -%>
         The IETF status of this document can be found at <eref target="https://datatracker.ietf.org/doc/<%=docname.rpartition('-')[0]%>/"/>
```

Comment:
Strictly speaking, the IETF status of the latest submitted version is found there.
Not sure this can be expressed in a concise way.

Reply:
... and not all I-D documents have IETF status.
Pushed 11b6f51 for that.